test/YoloDotNet.Test.Common/SharedConfig.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace YoloDotNet.Benchmarks
+{
+    using System;
+    using System.IO;
+
+    using YoloDotNet.Enums;
+    using YoloDotNet.Test.Common.Enums;
+
+    public static class SharedConfig
+    {
+        private const string ASSETS_FOLDER = @".\assets";
+        private const string BASE_MODELS = ASSETS_FOLDER + @"\models";
+        private const string BASE_MEDIA = ASSETS_FOLDER + @"\media";
+
+        public static string GetTestModel(ModelType modelType) => modelType switch
+        {
+            ModelType.Classification => Path.Combine(BASE_MODELS, "yolov8s-cls.onnx"),
+            ModelType.ObjectDetection => Path.Combine(BASE_MODELS, "yolov8s.onnx"),
+            ModelType.ObbDetection => Path.Combine(BASE_MODELS, "yolov8s-obb.onnx"),
+            ModelType.Segmentation => Path.Combine(BASE_MODELS, "yolov8s-seg.onnx"),
+            ModelType.PoseEstimation => Path.Combine(BASE_MODELS, "yolov8s-pose.onnx"),
+            _ => throw new ArgumentException("Unknown modeltype.")
+        };
+
+        public static string GetTestImage(ImageType imageType) => imageType switch
+        {
+            ImageType.Hummingbird => Path.Combine(BASE_MEDIA, "hummingbird.jpg"),
+            ImageType.Street => Path.Combine(BASE_MEDIA, "street.jpg"),
+            ImageType.People => Path.Combine(BASE_MEDIA, "people.jpg"),
+            ImageType.Crosswalk => Path.Combine(BASE_MEDIA, "crosswalk.jpg"),
+            ImageType.Island => Path.Combine(BASE_MEDIA, "island.jpg"),
+            _ => throw new ArgumentException("Unknown modeltype.")
+        };
+
+        public static string GetTestImage(string imageName) => Path.Combine(BASE_MEDIA, imageName);
```

Comment:
The `Assets` folder and its contents is currently not visible to the user in the Solution Explorer. The `Assets` folder could be moved into the `YoloDotNet.Test.Common` for a more centralized configuration. To get access to the images and models, `SharedConfig.cs` could then be updated to something like this:
```csharp
private const string _projectName = nameof(YoloDotNet);

private static readonly string _dir = AppDomain.CurrentDomain.BaseDirectory;
private static readonly string _baseDir = _dir[0..(_dir.IndexOf(_projectName) + _projectName.Length)];

private static readonly string _assetsFolder = Path.Combine(_baseDir, @"test\YoloDotNet.Test.Common\Assets");
private static readonly string _baseModels = Path.Combine(_assetsFolder, "Models");
private static readonly string _baseMedia = Path.Combine(_assetsFolder, "Media");

public static string GetTestModel(ModelType modelType) => modelType switch
{
    ModelType.Classification => Path.Combine(_baseModels, "yolov8s-cls.onnx"),
    ModelType.ObjectDetection => Path.Combine(_baseModels, "yolov8s.onnx"),
    ModelType.ObbDetection => Path.Combine(_baseModels, "yolov8s-obb.onnx"),
    ModelType.Segmentation => Path.Combine(_baseModels, "yolov8s-seg.onnx"),
    ModelType.PoseEstimation => Path.Combine(_baseModels, "yolov8s-pose.onnx"),
    _ => throw new ArgumentException("Unknown modeltype.")
};

public static string GetTestImage(ImageType imageType) => imageType switch
{
    ImageType.Hummingbird => Path.Combine(_baseMedia, "hummingbird.jpg"),
    ImageType.Street => Path.Combine(_baseMedia, "street.jpg"),
    ImageType.People => Path.Combine(_baseMedia, "people.jpg"),
    ImageType.Crosswalk => Path.Combine(_baseMedia, "crosswalk.jpg"),
    ImageType.Island => Path.Combine(_baseMedia, "island.jpg"),
    _ => throw new ArgumentException("Unknown test image.")
};

public static string GetTestImage(string imageName) => Path.Combine(_baseMedia, imageName);
```